### PR TITLE
Fix shopify readme

### PR DESCRIPTION
**Pull request by floelhoeffel**

### CODE REVIEW

File: samples/shopify/README.md [hunk 1]
```diff
@@ -1,53 +1,71 @@
...
+To install the UI Extension:
 
-## Ready to Use
+```bash
+contentful extension create
+```
 
-![figure](./shopify-object.png "Contentful Shopify Product UI Extension json object")
\ No newline at end of file
+To update the UI Extension:
```
Comment (stefanjudis) [inline samples/shopify/README.md]: Consistency nitpick: A new line is missing between paragraph and code snippet.

File: samples/shopify/README.md [hunk 2]
```diff
@@ -1,53 +1,71 @@
...
-Open `.env` in a editor of your liking and add your Contentful space ID, and management token. [Learn how to obtain a token](https://www.contentful.com/developers/docs/references/authentication/#getting-an-oauth-token).
+## Usage
 
-Load environment variables
+Make sure you have the Contentful CLI installed:
 
-```sh
-source .env
+```bash
+npm install contentful-cli -g
```
Comment (stefanjudis) [inline samples/shopify/README.md]: It's a bummer that we have to recommend global installation of the cli... as far as I see there is no way to hand in the cma-token via parameter anymore? :( @Khaledgarbaya

Comment (Khaledgarbaya) [reply samples/shopify/README.md]: you should be able to pass the accessToken for all command but it is not the case for the extensions command, @stefanjudis can you create an issue there

File: samples/shopify/README.md (hunk 1 above)
Comment (floelhoeffel) [reply samples/shopify/README.md]: done

**Comment by floelhoeffel** [comment]
@stefanjudis PTAL

File: samples/shopify/README.md (hunk 2 above)
Comment (stefanjudis) [reply samples/shopify/README.md]: Done. https://github.com/contentful/contentful-cli/issues/41

**Pull request merged**